src/scgpt/binning/script.py

```diff
@@ -0,0 +1,120 @@
+import mudata as mu
+import numpy as np
+from scanpy.get import _get_obs_rep
+from scipy.sparse import csr_matrix
+import warnings
+
+## VIASH START
+par = {
+    "input": "resources_test/scgpt/test_resources/Kim2020_Lung_subset.h5mu",
+    "output": "resources_test/scgpt/test_resources/Kim2020_Lung_subset_binned_sparse.h5mu",
+    "modality": "rna",
+    "input_layer": "X",
+    "binned_layer": "X_binned",
+    "n_input_bins": 51,
+    "output_compression": None
+}
+## VIASH END
+np.random.seed(0)
```

Comment:
I don't think we need so set it in production, but we might want it for testing. Could you create a parameter for the seed?